### ISSUE TITLE
Is there a way to connect our mailing list to GH, so that issues in GH are sent to the list?

### ISSUE THREAD

**Issue by dlemire60** [opencybersecurityalliance/PACE#32]

**Comment by claurau** [comment]
Yes, that should be possible. Let me know if we should set this up.

**Comment by sparrell** [comment]
I am against tying the github mail list to the PACE mail list. There is already a mailing list for the repo. I think anyone who wants can join this mailing list instead of sending all the git comments and commits to the general list to people who don't want it and can't unsubscribe. Instead they'll unsubscribe from the pace mailing list which is not what we want.

**Comment by adammontville** [comment]
Duncan (and all),

Certainly open to discussion. The reason tying the two together could be helpful is that there may be a subset of our community without GitHub accounts and with no driver to establish one, but they would still like to opine on certain issues, such as those surrounding governance. If we don’t believe we have this subset in our community, then we don’t need to tie the two together - we can rely on people choosing to watch the GitHub repo.

To that end, is there anyone on this list who would prefer not to have a GitHub account and would like to still be aware of what is going on in the issues? 

Kind regards,

Adam

PS: This approach is a carryover from when I co-chaired an IETF working group, and all discussions were required to be on the list.

**Comment by dlemire60** [comment]
I think my ideal solution would be to keep issue discussions on GH without adding them to the PACE mail list, but it might be advantageous if we could somehow propagate issue creation to the mail list. That way list members who don’t watch the repo at least get some notification of new issues while list members who are watching the repo won’t have to deal with duplicate message from GH and the mail list when comments are added to an issue.
